CONTRIBUTING.md

```diff
@@ -99,6 +99,9 @@ Your commit will be converted into a patch that you can then PR into Paper.
 
 Modifying previous patches is a bit more complex:
 
+> ! When modifying patches, you should be operating on the sources directory
```

Comment:
I feel like this is covered in the section above?

Reply:
Hmm, it's stated there, but its still not clear (to me at least) that the same should be followed for modification.